js/read-csv.js

```diff
@@ -32,6 +32,26 @@ function processData(csv) {
 	drawOutput(lines);
 }
 
+//if your csv file contains the column names as the first line
+function processDataAsObj(csv){
+    var allTextLines = csv.split(/\r\n|\n/);
+    var lines = [];
+	
+    //first line of csv
+    var keys = allTextLines.shift().split(',');
+	
+    while (allTextLines.length) {
+        var arr = allTextLines.shift().split(',');
+        var obj = {};
+        for(var i = 0; i < keys.length; i++){
+            obj[keys[i]] = arr[i];
+	}
+        lines.push(obj);
+    }
+        console.log(lines);
+	//drawOutput(lines);
```

Comment:
@Seanmcdon I think you should call `drawOutputAsObj` here and remove the commented line